cli/src/lib/validators/validatePackageExports.js

```diff
@@ -31,21 +31,20 @@ module.exports.validatePackageExports = async (
     const baseDir = path.dirname(paths.package)
 
     let valid = true
-    const entrypointBasename = normalizeExtension(
-        path.basename(config.entryPoints.lib)
-    )
+    const entrypointBasename = normalizeExtension(config.entryPoints.lib)
```

Comment:
Also - this variable is no longer the `basename` so it should be renamed (just `entrypoint` should be fine)

Reply:
Dropped the interim variable in: 2dda1d9